modules/apps/portal-cache/portal-cache-impl/src/test/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImplTest.java

```diff
@@ -101,6 +101,8 @@ public void setUp() {
 	public void testNotifyPortalCacheRemovedPortalCacheName() {
 		EntityCacheImpl entityCacheImpl = new EntityCacheImpl();
 
+		BundleContext bundleContext = Mockito.mock(BundleContext.class);
```

Comment:
@GabrielHenriquedev instead of using `Mockito`, use `SystemBundleUtil.getBundleContext();`